qipr_approver/static/qipr/js/app.js

```diff
@@ -83,11 +83,16 @@
         window.typingTimer = setTimeout(doneTyping, doneTypingInterval, node);
     };
 
+    var getHostnameSuffix = function () {
+        return window.location.hostname === 'qipr.ctsi.ufl.edu' ? '/approver' : '';
```

Comment:
Is there a way to handle both staging and production requests dynamically? stage is stage.qipr.ctsi.ufl.edu and prod is just qipr.ctsi.ufl.edu